### DISCUSSION TITLE
Enable plugin per port

### DISCUSSION THREAD

**Discussion by Straith** [synman/Octoprint-Bettergrblsupport#86] [Ideas]
I've replicated my octoprint directory following the directions found here (https://www.youtube.com/watch?v=GEgukBiLnmM) and successfully have a second instance for controlling a second device. My intention is to run my CR10S on one instance and my 3018 Prover on the second instance. With BetterGRBLSupport installed, I get the GRBL UI on both instances. If I disable the plugin, it's back to normal for both. Is there a way to disable your plugin for my CR10S but keep it for my 3018 Prover? With the price of Raspberry's at scalper levels I am hoping to make this work.

**Comment by synman** [comment]
hmm... it doesn't sound like you truly have unique instances running if they are sharing configs.
